### PR TITLE
Sprint12 04 a44370 db lineage override restore

### PR DESCRIPTION
**Pull request by lopaka**
[db] added restore recipes under timestamp_override and lineage_override attributes

### CODE REVIEW

File: cookbooks/db/metadata.rb [hunk 1]
```diff
@@ -179,26 +179,30 @@
   ]
 
 attribute "db/backup/lineage_override",
-  :display_name => "Database Backup Lineage Override",
+  :display_name => "Database Restore Lineage Override",
   :description => "If defined, this will override the input defined for 'Backup Lineage' (db/backup/lineage) so that you can restore the database from another backup that has as different lineage name. The most recently completed snapshots will be used unless a specific timestamp value is specified for 'Restore Timestamp Override' (db/backup/timestamp_override). Although this input allows you to restore from a different set of snapshots, subsequent backups will use 'Backup Lineage' to name the snapshots. Be sure to remove the 'Backup Lineage Override' input after the new master is operational.",
```
Comment (ekargard) [inline cookbooks/db/metadata.rb:183]: This line needs to be split so it can be viewed without scrolling when checking a pull request.

**Comment by ekargard** [comment]
How did sys_dns metadata update make it into this pull request?  Is the master branch of sys_dns metadata.rb out of sync with metadata.json?

File: cookbooks/db/metadata.rb (hunk 1 above)
Comment (lopaka) [inline cookbooks/db/metadata.rb:183]: metadata is not the best place to do this.

**Comment by douglaswth** [comment]
This looks fine.

**Pull request merged**